### PR TITLE
HTTP benchmark task

### PR DESCRIPTION
**Pull request by nikoncode**
FYI @kortelyov

### CODE REVIEW

**Review by kortelyov — COMMENTED** [review]
Main task still doesn't complete

File: http_benchmark/main.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	var group sync.WaitGroup
+	client := http.Client{
+		Timeout: timeout,
+	}
+	var sum, timeouts, max, min, count int64
+	max, min = math.MinInt64, math.MaxInt64
+
+	for i := 0; i < requests; i++ {
+		group.Add(1)
+		go func() {
```
Comment (kortelyov) [inline http_benchmark/main.go]: don't use anonymous functions

File: http_benchmark/main.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			defer group.Done()
+
+			start := time.Now()
+			_, err := client.Get(url)
```
Comment (kortelyov) [inline http_benchmark/main.go]: where is err check?

**Review by kortelyov — CHANGES_REQUESTED** [review]
default output
{min:665978007 max:1193735215 avg:9.6014168299e+08 timeouts:0}
what does it mean ?

Also I wrote about some URLs (1-...), and we should got separated stats for all of them

File: http_benchmark/main.go (hunk 1 above)
Comment (kortelyov) [reply http_benchmark/main.go]: also add wg pointer to the goroutine's args

**Comment by nikoncode** [comment]
@kortelyov please review again. No more mutexes.

**Review by kortelyov — APPROVED** [review]

**Pull request merged**